.gitignore

```diff
@@ -19,6 +19,7 @@ public/.htaccess
 public/assets
 public/images/.dpg_pool
 tmp/
+workflow_service.log*
```

Comment:
The `.gitignore` already contains `log/*.log` - is there a setting that can be changed to put the `workflow_service.log` file into the `log` path?